### DISCUSSION TITLE
Navbar modification

### DISCUSSION THREAD

**Discussion by MARLEY-TECH** [smaranjitghose/awesome-portfolio-websites#1317] [General]
Hello, thank you for this awesome portfolio. Please how can we modify the Navbar?

**Comment by anushbhatia** [comment]
https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/assets/js/app.js you can use this

**Reply by MARLEY-TECH** [reply]
Thank you i can now modify it.

**Reply by subhankar-pahi-2005** [reply]
Thank you I also suffering with how can i modify nav bar
